devel/gh/Portfile

```diff
@@ -35,5 +35,9 @@ destroot {
 
     set docdir ${prefix}/share/doc/${name}
     xinstall -d ${destroot}${docdir}
-    xinstall -m 0644 -W ${worksrcpath} LICENSE README.md CHANGELOG.md ${destroot}${docdir}
+    xinstall -m 0644 -W ${worksrcpath} LICENSE ${destroot}${docdir}
 }
+
+notes "
+For help on using the Github CLI, refer to: https://cli.github.com/manual/
```

Comment:
I think our users would check the homepage for usage, so there's no need for notes here.

Reply:
OK. Removed.